action.yaml

```diff
@@ -260,7 +274,7 @@ runs:
       shell: bash
       run: |
         # Run 'trunk check' on pull request
-        ${GITHUB_ACTION_PATH}/pull_request.sh
+        timeout ${{ inputs.timeout-seconds }} ${GITHUB_ACTION_PATH}/pull_request.sh
```

Comment:
Is it possible to omit the timeout at all if it is not set. I'm not sure if we can assume this command exists on every self-hosted runner.

Reply:
I can do that!